Library/ViewModels/CommentsViewModel.swift

```diff
@@ -235,12 +232,26 @@ public final class CommentsViewModel: CommentsViewModelType,
     )
     .takePairWhen(commentComposerDidSubmitText)
     .map(unpack)
-    .flatMap(postCommentProducer)
+    .flatMap(.concurrent(limit: concurrentCommentLimit), postCommentProducer)
```

Comment:
The `.concurrent(limit:)` strategy here allows use to schedule multiple `SignalProducer`s on the queue simultaneously, I have the limit set to 3 because it seems like a good amount but it can likely be quite a bit higher. We need to bear in mind that it will spawn multiple threads and one would not want this to overload the system.